install.sh

```diff
@@ -1,34 +1,49 @@
 #!/bin/sh
 set -e
-if [ "$1" != "" ]; then
-	if [ -d "$1" ]; then
-		mkdir -p $1/tools/agbcc
-		mkdir -p $1/tools/agbcc/bin
-		mkdir -p $1/tools/agbcc/include
-		mkdir -p $1/tools/agbcc/lib
-		cp agbcc $1/tools/agbcc/bin/
-		cp old_agbcc $1/tools/agbcc/bin/
-		cp agbcc_arm $1/tools/agbcc/bin/
-		cp -R libc/include $1/tools/agbcc/ #drop include, because we don't want include/include
-		cp ginclude/* $1/tools/agbcc/include/
-		cp libgcc.a $1/tools/agbcc/lib/
-		cp libc.a $1/tools/agbcc/lib/
-		echo "agbcc successfully installed!"
+
+if [ "$1" = "-h" ]; then
```

Comment:
```suggestion
if [ "$1" = -h -o "$1" = --help ]; then
```
When parsing arguments as flags, it's appropriate to have a `--` flag to end the list of argument flags, after which the rest is interpreted as positional arguments, and fail when an unknown flag is passed. That said, this is only for -h and I doubt anyone would install to "--help", but keep it in mind and consider using `getopt` in the future.